src/storage/copier/string_column_chunk.cpp

```diff
@@ -49,6 +49,19 @@ void StringColumnChunk::resetToEmpty() {
     overflowCursor.resetValue();
 }
 
+void StringColumnChunk::append(common::ValueVector* vector, common::offset_t startPosInChunk) {
+    assert(vector->dataType.getPhysicalType() == PhysicalTypeID::STRING);
+    ColumnChunk::copyVectorToBuffer(vector, startPosInChunk);
+    auto stringsToResetOverflow = (ku_string_t*)(buffer.get() + startPosInChunk * numBytesPerValue);
```

Comment:
What does `reset` mean in `stringsToResetOverflow`? I guess you mean `set`? Consider renaming it to `stringsToSet`.